### DISCUSSION TITLE
Lookup for errors and info messages

### DISCUSSION THREAD

**Discussion by Vyeche** [vrtmrz/obsidian-livesync#108] [Q&A]
Hi, 

Is there a document explaining the errors and info messages?
I noticed these errors why I ran the sanity check. 

- _design/replicate missing children
- xyz.md missing child(ren)

What should I do in these cases?

Thanks & kind regards,
Vye

**Comment by vrtmrz** [comment]
Thank you for making this topic!
These messages were written in the early. So terrible English.
`Children` is now called `Chunks`. But a while ago, it was called `leaf` or `piece`.
It is in [Tech info](https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/tech_info.md)  but so outdated.

I will write new docs to make it better! And messages should be clear.

P.S. _design/replicate has no children by nature. I have a bad feeling about this...

**Reply by Vyeche** [reply]
Cool, so if a file is missing chunks what does that mean? Do I need to act?

**Reply by vrtmrz** [reply]
Usually, they will be created again if any changes are on the file (on xyz.md) or are replicated if any other device has.
If we want to fix them immediately, we can restore chunks by `Restore from file` in the `🚑️ Corrupted or missing data` on the setting dialogue.
(This will be shown only corruption has been detected)

**Reply by Vyeche** [reply]
Great explanation, thank you. I added it to my notes. 😊
